indra/newview/llreflectionmap.cpp

```diff
@@ -166,7 +166,7 @@ void LLReflectionMap::autoAdjustOrigin()
             
         }
     }
-    else if (mViewerObject)
+    else if (mViewerObject && !mViewerObject->isDead())
```

Comment:
More `isDead` checks is likely good, and could probably be propagated to other `mViewerObject` null checks in this class, such as `LLReflectionMap::isRelevant`